src/projects/republic/HelpLink.tsx

```diff
@@ -0,0 +1,14 @@
+export function HelpLink() {
+  return (
+    <div className="mr-4 flex grow flex-row items-center justify-end gap-2">
+      <a
```

Comment:
Wellicht `target="_blank"` toevoegen zodat de helppagina in een nieuw tabblad wordt geopend?

Reply:
Fixed in [ba3c4d3](https://github.com/knaw-huc/textannoviz/pull/282/commits/ba3c4d3115034f3ea1bb6c6054e36fe93899c640)